tldw_Server_API/app/main.py

```
@@ -2799,6 +2799,26 @@ async def _ephemeral_cleanup_loop():
+            )
+
+            recipe_run_jobs_task = await start_recipe_run_jobs_worker()
+            if recipe_run_jobs_task:
+                logger.info("Evaluation recipe-run Jobs worker started")
+            else:
+                logger.info(
+                    "Evaluation recipe-run Jobs worker disabled "
+                    "(EVALUATIONS_RECIPE_RUN_JOBS_WORKER_ENABLED != true)"
+                )
+    except _STARTUP_GUARD_EXCEPTIONS as e:
+        logger.warning(f"Failed to start evaluation recipe-run Jobs worker: {e}")
```

Ruff (0.15.7)

[warning] Logging statement uses f-string

(G004)